src/Features/CSharp/Portable/ReplacePropertyWithMethods/CSharpReplacePropertyWithMethodsService.cs

```diff
@@ -98,20 +98,16 @@ public override SyntaxNode GetPropertyDeclaration(SyntaxToken token)
                     documentOptions, parseOptions, 
                     generator, propertyDeclaration, propertyBackingField,
                     getMethod, desiredGetMethodName,
-                    copyLeadingTrivia: true,
                     cancellationToken: cancellationToken));
             }
 
             var setMethod = property.SetMethod;
             if (setMethod != null)
             {
-                // Set-method only gets the leading trivia of the property if we didn't copy
-                // that trivia to the get-method.
```

Comment:
We need some amount of logic to ensure that we don't copy all trivia from the method to both the getter and the setter.  For example of there is a #pp directive on hte method, then it should only be copied to the first generated member.